src/__tests__/cat-list.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { CatsList } from '../components/cat-list/cats-list';
+import { mockCats } from './mocks/cats.mock';
+import { messages } from '../sources/messages';
+
+describe('Cat list', () => {
+  it('should show message if no cats found', () => {
+    render(<CatsList cats={[]} />);
+    const message = screen.getByText(messages.noCatsFound);
+    expect(message).toBeInTheDocument();
+  });
```

Comment:
fixing react warning about state updates not being included in act(...):
```suggestion
  it('should show message if no cats found', () => {
    render(<CatsList cats={[]} />);
    waitFor(() => {
      const message = screen.getByText(messages.noCatsFound);
      expect(message).toBeInTheDocument();
    });
  });
```